kms/api-client/snippets.py

```diff
@@ -61,70 +61,70 @@ def create_cryptokey(project_id, location, keyring, cryptokey):
 
 
 # [START kms_encrypt]
-def encrypt(project_id, location, keyring, cryptokey, plaintext_file_name,
-            encrypted_file_name):
-    """Encrypts data from a plaintext_file_name using the provided CryptoKey
-    and saves it to an encrypted_file_name so it can only be recovered with a
-    call to decrypt."""
+def encrypt(project_id, location_id, key_ring_id, crypto_key_id,
+            plaintext_file_name, ciphertext_file_name):
+    """Encrypts data from plaintext_file_name using the provided CryptoKey and
+    saves it to ciphertext_file_name so it can only be recovered with a call to
+    decrypt.
+    """
 
     # Creates an API client for the KMS API.
     kms_client = googleapiclient.discovery.build('cloudkms', 'v1')
 
     # The resource name of the CryptoKey.
     name = 'projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}'.format(
-        project_id, location, keyring, cryptokey)
+        project_id, location_id, key_ring_id, crypto_key_id)
 
-    # Read text from the input file.
+    # Read data from the input file.
     with io.open(plaintext_file_name, 'rb') as plaintext_file:
         plaintext = plaintext_file.read()
 
-    # Use the KMS API to encrypt the text.
-    cryptokeys = kms_client.projects().locations().keyRings().cryptoKeys()
-    request = cryptokeys.encrypt(
+    # Use the KMS API to encrypt the data.
+    cryptoKeys = kms_client.projects().locations().keyRings().cryptoKeys()
```

Comment:
Fixed. Sorry, I didn't intend to go against the style guide, just an oversight on my part.